src/test/scala/is/hail/utils/RowIntervalSuite.scala

```diff
@@ -115,6 +115,12 @@ class RowIntervalSuite extends TestNGSuite {
     assert(eord.lteqWithOverlap(0)(
       IntervalEndpoint(Row(0, 3), -1), IntervalEndpoint(Row(1, 2), -1)
     ))
+    assert(!eord.lteqWithOverlap(-1)(
```

Comment:
What does overlap == -1 signify?

Reply:
The semantics of `lteqWithOverlap(n)(x, y)` is that for any rows `a, b` with `a<x` and `y<b`, `a` and `b` can agree in a prefix of length at most `n`. As any rows agree in a prefix of length 0, `lteqWithOverlap(-1)` always returns false for any `x, y`. This came up in an edge case somewhere, so I added a test to cover it.